doc/userguide/lua/lua-functions.rst

```diff
@@ -481,71 +481,6 @@ Initialize with:
       return needs
   end
 
-SshGetServerProtoVersion
```

Comment:
Is that right ? We remove the old API @jasonish , correct ?

Reply:
Yes, remove the old API.